career_development/ladder.md

```diff
@@ -0,0 +1,114 @@
+# Abtion Dev Career Ladder
+
+## High Level Company Values
+1. Bonded by Love for Product
+1. Continuous Delivery
+1. Customer Comes First
+1. Heavily Team Oriented
+1. High Employee Retention
+1. High Quality Code Base
+1. Open Communication 
+1. Pair-Programs
+
+## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
+
+### Level 1
+
+#### simple 
+-[ ] is working on naming variables and learning to extract methods
+	
+#### test-driven
+-[ ] runs all tests before commiting, can understand where expectations are not met and fix some with the help of more senior developers
+	
+#### iterative
+-[ ] understands red-green-refactor
+	
+#### communication
+-[ ] updates asana, comes prepared for standup, uses time tracking tools and vacation notifications
+	
+#### respectful teammate
+-[ ] on time for meetings, is enthusiastic
+	
+#### hard skills
+-[ ] can use git, linters, makes clear prs
```

Comment:
There's probably a better word than `hard` for this section.

Reply:
"can use git" is vague.

Reply:
@GeoffAbtion What do you think about hard -> technical? Also, how about 'is able to setup and make commits to github from the commandline'? Feel free to add commits with that.

Reply:
I think it's a bit exclusive to couple it with a CLI since there's loads of GUI-based tools for Git. One could probably be very good at Git without using it via a CLI.

Suggestion to make the "can use Git"-part more explicit:
_Knows about the three main sections of a Git project(working, staging, repository) as well as the most common commands(commit, pull, push)_
The wording can be improved, but it's the kind of explicitness I believe would be beneficial.

I agree that being able to make clear PRs should be on a higher level. Sure it's easy to make PRs, but finding good, clean ones is sometimes difficult.

Using "technical skills" instead of "hard skills" is much better. The usage of "hard skills" also implies that we use the "soft skills" term which I find a bit unfortunate - the "soft skills" are the hardest ones to master and the wording undervalues them.